### ISSUE TITLE
(a11y) FocusTrapZone does not work well with ChoiceGroup

### ISSUE THREAD

**Issue by wudicheng123** [OfficeDev/office-ui-fabric-react#5637]
### Bug Report 
 
- __Package version(s)__: 6.26.0
- __Browser and OS versions__: Chrome 67.0.3396.99
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
No

Requested priority: (Blocking, High, Normal, Low) 
High

Products/sites affected: (if applicable)
 
#### Describe the issue:
 
If the last element of the ChoiceGroup is not tab enabled, the focus will jump out of the element  
 
![image](https://user-images.githubusercontent.com/9521662/42971546-9c47023e-8b61-11e8-8f6e-c5f74338f031.png)

#### Actual behavior:
tab focus jumps outside of the element 
 
#### Expected behavior:
tab focus should remain inside of the element

**Comment by natalieethell** [comment]
Thanks for submitting this issue @wudicheng123! This is a dupe of #5160. @JasonGore is working on a fix in this PR: https://github.com/OfficeDev/office-ui-fabric-react/pull/5842. I'll close this issue once the changes are merged!
